pkgs/applications/networking/browsers/firefox/default.nix

```diff
@@ -64,7 +65,6 @@ stdenv.mkDerivation rec {
       "--disable-necko-wifi" # maybe we want to enable this at some point
       "--disable-installer"
       "--disable-updater"
-      "--disable-pulseaudio"
```

Comment:
That's what it does -- I made it available in firefoxWrapper with `LD_LIBRARY_PATH` if `config.pulseaudio` is set.